tests/develop/max_nesting_level-003.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
```

Comment:
Note: this test is still valid, as it demonstrates that a nesting level of 10000 is ignored by the defaults